Fail tests test_fft.TestFFT on Ubuntu 20.04

Hello,
I'm starting to use bifrost and I'm happily starting with the tutorials.
However, just out of curiosity, to check my install, I run `make test`.
Many passed but `FAILED (failures=3, skipped=4)`

FAIL: test_c2r_1D (test_fft.TestFFT)
Traceback (most recent call last):
  File "/home/cedric/tmp/bifrost/test/test_fft.py", line 206, in test_c2r_1D
    self.run_test_c2r(self.shape1D, [0])
  File "/home/cedric/tmp/bifrost/test/test_fft.py", line 148, in run_test_c2r
    self.run_test_c2r_impl(shape, axes)
  File "/home/cedric/tmp/bifrost/test/test_fft.py", line 141, in run_test_c2r_impl
    compare(odata.copy('system'), known_result)
  File "/home/cedric/tmp/bifrost/test/test_fft.py", line 51, in compare
    np.testing.assert_allclose(result, gold, rtol=RTOL, atol=MTOL * absmean)
  File "/home/cedric/anaconda3/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 1528, in assert_allclose
    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
  File "/home/cedric/anaconda3/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 842, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.1, atol=0.00462357
Mismatched elements: 1 / 16777216 (5.96e-06%)
Max absolute difference: 0.01639435
Max relative difference: 1.93911746

FAIL: test_c2r_2D (test_fft.TestFFT)
AssertionError: 
Not equal to tolerance rtol=0.1, atol=0.00231149
Mismatched elements: 4186048 / 4194304 (99.8%)
Max absolute difference: 492620.22392237
Max relative difference: 39550830.05759069

FAIL: test_c2r_3D (test_fft.TestFFT)
AssertionError: 
Not equal to tolerance rtol=0.1, atol=0.00163087
Mismatched elements: 2080441 / 2097152 (99.2%)
Max absolute difference: 88917.37869481
Max relative difference: 6823521.1182115

This was tested on Ubuntu 20.04.4 LTS with Python 3.8.8.

Tell me if I can help.
Regards,
Cedric

--- a/configure
+++ b/configure
@@ -26801,7 +26801,7 @@ fi
 
 if test x$HAVE_CUDA = x1
 then :
-  cufft_c2r_modifies=$( echo $CUDA_VERSION | ${GREP} -e "10\.[1-2]" -e "11\.[0-6]" )
+  cufft_c2r_modifies=$( echo $CUDA_VERSION | ${GREP} -e "10\.[1-2]" -e "11\.[0-7]" )
        if test "$cufft_c2r_modifies" != ""; then
          echo ""
          { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: This version of cuFFT may have unexpected behavior for complex-to-real transforms" >&5

--- a/configure.ac
+++ b/configure.ac
@@ -346,7 +346,7 @@ AC_OUTPUT
 #
 
 AS_IF([test x$HAVE_CUDA = x1],
-      [cufft_c2r_modifies=$( echo $CUDA_VERSION | ${GREP} -e "10\.[[1-2]]" -e "11\.[[0-6]]" )
+      [cufft_c2r_modifies=$( echo $CUDA_VERSION | ${GREP} -e "10\.[[1-2]]" -e "11\.[[0-7]]" )
        if test "$cufft_c2r_modifies" != ""; then
          echo ""
          AC_MSG_WARN(This version of cuFFT may have unexpected behavior for complex-to-real transforms)

--- a/test/test_fft.py
+++ b/test/test_fft.py
@@ -51,7 +51,7 @@ def compare(result, gold):
     np.testing.assert_allclose(result, gold, rtol=RTOL, atol=MTOL * absmean)
 
 def cudaExpectedFailure(testcase):
-    if BF_CUDA_VERSION < 10.1 or BF_CUDA_VERSION > 11.6:
+    if BF_CUDA_VERSION < 10.1 or BF_CUDA_VERSION > 11.7:
         return testcase
     return unittest.expectedFailure(testcase)
 

